trace/opencensus/trace/ext/django/middleware.py

```diff
@@ -112,6 +112,10 @@ def process_request(self, request):
                 reporter=self._reporter(),
                 propagator=propagator)
             setattr(_thread_locals, TRACER_THREAD_LOCAL_KEY, tracer)
+
+            if not tracer.enabled:
```

Comment:
I plan to add a no-op tracer to handle the non-sampled cases, instead of checking whether it's sampled or not every time it calls some function in tracer. Will submit a separate PR for that.